eth-keystore/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "eth-keystore"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+zeroize = "1.6.0"
+anyhow = { workspace = true }
+ethers = { workspace = true, optional = true }
+serde = { workspace = true }
+serde_json = { workspace = true }
+hex = "0.4.3"
```

Comment:
Should we make this dependency part of the workspace? IIRC we are using it in other crates also.